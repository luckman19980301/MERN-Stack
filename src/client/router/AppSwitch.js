import React from 'react';
import { Switch, Route } from 'react-router-dom';
import Hello from '../components/Hello';

const AppSwitch = (props) => (
    <Switch>
        <Route path='/' component={ Hello } exact />
    </Switch>
);

export default AppSwitch;
