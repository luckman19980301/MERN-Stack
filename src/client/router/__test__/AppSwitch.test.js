import React from 'react';
import { shallow } from 'enzyme';
import AppSwitch from '../appSwitch.jsx';

describe('AppSwitch component', () => {
  const wrapper = shallow(<AppSwitch />);

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
