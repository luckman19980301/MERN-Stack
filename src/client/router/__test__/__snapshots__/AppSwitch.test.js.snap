// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSwitch component renders correctly 1`] = `
"<Switch>
  <Route exact={true} path=\\"/\\" component={[Function: Hello]} />
  <Route exact={true} path=\\"/test\\" component={[Function: component]} />
</Switch>"
`;
